src/components/StackedBarChart/drawD3Chart.tsx

```diff
@@ -125,26 +129,7 @@ export function drawD3Chart(
   // style y and x axis
   const yAxis = axisLeft(yScale).ticks(yTicks);
 
-  const xAxis = axisBottom<Date>(xScale)
-    .ticks(6)
-    .tickFormat((tickDate: Date): string => {
-      // get sysdate for x Axis
-      const today = new Date();
-
-      const dateIsToday =
-        tickDate.getFullYear() === today.getFullYear() &&
-        tickDate.getMonth() === today.getMonth() &&
-        tickDate.getDate() === today.getDate();
-
-      if (dateIsToday) {
-        return 'Heute';
-      }
-      // let formattedTime = formatter(d);
-      let daysBack = today.getTime() - tickDate.getTime();
-      daysBack = daysBack / (1000 * 3600 * 24);
-      daysBack = Math.round(daysBack);
-      return daysBack === 1 ? 'Gestern' : `Vor ${daysBack} Tagen`;
-    });
+  const xAxis = axisBottom<Date>(xScale).ticks(5).tickSizeOuter(0);
```

Comment:
If I get it right here, you got rid of our custom formating. Was it discussed with @julizet ? And also the formating here is still in English when it should be German.